src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,46 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<K, V> {
+    private K firstValue;
+    private V secondValue;
 
+    private Pair(K firstValue, V secondValue) {
+        this.firstValue = firstValue;
+        this.secondValue = secondValue;
+    }
+
+    public static <K, V> Pair of(K firstValue, V secondValue) {
```

Comment:
```suggestion
    public static <K, V> Pair<K, V> of(K firstValue, V secondValue) {
```
We creating Pair object here and returning it, so we should parametrize the object which returns